e2e/crm4/provider/not-prison-matter.spec.js

```diff
@@ -0,0 +1,75 @@
+import { test, expect } from '@playwright/test';
```

Comment:
Given this includes both prison law and not prison law scenarios, and the common theme is that the total value is too low, I think maybe there's a better file name than "not-prison-matter"

Reply:
On second thought, I rather include caseworker side here. making this PR to draft again till then.